src/resources/views/crud/inc/edit_translation_notice.blade.php

```diff
@@ -0,0 +1,91 @@
+    @php
```

Comment:
This alert needs a little more work in terms of looks. The current design looks unpolished:
![CleanShot 2024-06-18 at 12 16 32@2x](https://github.com/Laravel-Backpack/CRUD/assets/1032474/515b3478-35a4-417a-b586-71c8eb315c9f)

I played around with it a bit, and got it to this, which I think is a little better, especially if instead of having a dropdown... we just list all the available languages, so the admin clicks on the language they want as fallback:
![CleanShot 2024-06-18 at 12 15 40@2x](https://github.com/Laravel-Backpack/CRUD/assets/1032474/bf1e3818-b458-432d-93cb-bb494f307c4b)

And then I thought... **why show this notice in a persistent bubble there... or a text that we want to standa out... when we have notification bubbles?** @pxpm what do you think if... instead of showing this on-page... we use JS to trigger a notification bubble that stays onscreen (if not dismissed), with this text? To me it sounds like:
- it would be consistent across themes, with zero effort
- it would stand out more than it does now (as a yellow warning / blue info... not sure which is better)
- it would save us from designing an element here

If you agree a notification bubble is the best solution here, @pxpm , please implement that. Otherwise, let's go with a simpler solution like my screenshot above (yes, the translation button would be to the left, even when the notice isn't there).